resources/zib/zib-AdministrationAgreement.xml

```diff
@@ -0,0 +1,274 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-AdministrationAgreement" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-AdministrationAgreement" />
+  <name value="ZibAdministrationAgreement" />
+  <title value="zib AdministrationAgreement" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An administration agreement is the use (or administering) instructions from the pharmacist to the patient (or their representative or administrator), whereby a medication agreement is structured at a concrete level." />
+  <purpose value="This MedicationDispense resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) ](https://zibs.nl/wiki/AdministrationAgreement-v1.0.3(2020EN))." />

```

Comment:
Not confirm the guidelines missing the English zib name 

This [resource type] resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) [English zib name] [version]([release])]([link to the English zib page on zibs.nl]).

Reply:
Thnx. Resloved.